README.md

```diff
@@ -58,27 +59,22 @@ You can omit either of the two directories, and Meson will substitute
 the current directory and autodetect what you mean. This allows you to
 do things like this:
 
-`cd source_root; mkdir builddir; cd builddir; meson ..`
-
-or
-
-`cd source_root; mkdir builddir; meson builddir`
+```zsh
```

Comment:
I think this should be `console` not `zsh`

Reply:
sorry about that making the quick fix.

Reply:
ok change made. But I accidentally forgot the [skip ci] in the commit message, so sorry.